### PR TITLE
Welcome page shuffles

### LINKED ISSUE #352: Update dandiset welcome page
- Make the stats component for the number of dandisets link to the public dandisets page
- Vertically swap the stats and search box

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #352.

### CODE REVIEW

File: web/src/views/HomeView/SingleStat.vue [hunk 1]
```diff
@@ -18,7 +18,14 @@
         class="pa-0"
         data-id="name"
       >
-        {{ name }}
+        <a
+          v-if="href"
+          :href="href"
+        >
+          {{ name }}</a>
```
Comment (AlmightyYakob) [inline web/src/views/HomeView/SingleStat.vue:25]:
```suggestion
          {{ name }}
        </a>
```

**Review by AlmightyYakob — APPROVED** [review]
LGTM just one style comment

File: web/src/views/HomeView/SingleStat.vue (hunk 1 above)
Comment (waxlamp) [reply web/src/views/HomeView/SingleStat.vue:25]: Can't do it. The linebreak is interpreted by the html renderer as a space, which makes the link look ugly (the underline for hyperlinks extends one space to the right).

Comment (AlmightyYakob) [reply web/src/views/HomeView/SingleStat.vue:25]: Oh, alright then carry on.

**Pull request merged**